dask/array/creation.py

```diff
@@ -188,14 +187,29 @@ def linspace(start, stop, num=50, chunks=None, dtype=None):
         The last value of the sequence.
     num : int, optional
         Number of samples to include in the returned dask array, including the
-        endpoints.
+        endpoints. Default is 50.
+    endpoint : bool, optional
+        If True, ``stop`` is the last sample. Otherwise, it is not included.
+        Default is True.
+
+        .. versionadded:: 0.18.2
```

Comment:
Are we planning on another patch release, @mrocklin?

Reply:
I don't know

Reply:
I'll remove the `versionadded` since we're unsure about the next release version